fastimer/src/schedule/mod.rs

```diff
@@ -26,70 +27,19 @@ pub use notify::*;
 mod simple;
 pub use simple::*;
 
-use crate::info;
-use crate::MakeDelay;
-
 mod select;
-
-use crate::schedule::select::select;
 use crate::schedule::select::Either;
+use crate::schedule::select::select;
 
-/// Base trait for shutdown-able scheduled actions.
-pub trait BaseAction: Send + 'static {
-    /// The name of the trait.
-    fn name(&self) -> &str;
-
-    /// Return a future that resolves when the action is shutdown.
-    ///
-    /// By default, this function returns a future that never resolves, i.e., the action will never
-    /// be shutdown.
-    fn is_shutdown(&self) -> impl Future<Output = ()> + Send {
```

Comment:
No shutdown in Action since lieftime is tricky to handle. Follow axum's and poem's practice to pass a shutdown future that resolved to signal the action to shutdown.

I'd prefer to use a closure for geenrate the future each time but FnMut would face the issue as described in https://docs.rs/backon/latest/backon/trait.RetryableWithContext.html#example. And AsyncFnMut would hit the issue in https://github.com/rust-lang/rust/issues/62290#issuecomment-2816492235.